crates/fluvio/src/config/tls.rs

```diff
@@ -169,6 +172,8 @@ pub struct TlsPaths {
     pub cert: PathBuf,
     /// Path to Certificate Authority certificate
     pub ca_cert: PathBuf,
+    /// Secret name while putting in kubernetes
+    pub secret_name: String,
```

Comment:
@sehz We do need it because we use `TlsPaths` and `TlsCerts` in `fluvio-cluster` package. We use make use of `From` trait to convert between them interchangeably. Let me know if it makes sense !

Reply:
Fixed. Moved it to only Kube related configs.